app/views/lookbook/previews/preview.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if @render_args[:component] %>
   <% if defined?(Phlex::SGML) && @render_args[:component].is_a?(Phlex::SGML) %>
-    <%= raw(@render_args[:component].call(view_context: self, &@render_args[:block])) %>
+    <%= render(@render_args[:component]) { @render_args[:component].instance_eval(&@render_args[:block]) } %>
```

Comment:
Just a stylistic thing, I would make this use `instance_exec` since it's working with a block rather than a string. You could also pass the component instance to the instance_exec for cases where you want to pick up the yielded component.

Otherwise, I think this makes a lot of sense.

Reply:
Nice. I didn't know the instance_exec. It makes sense.